spec/unit/blockscore/collection/member_spec.rb

```diff
@@ -1,7 +1,32 @@
 module BlockScore
   class Collection
```

Comment:
@mbj Let me see if I understand:

Instead of layering the module, class references this should be:

`RSpec.describe BlockScore::Collection::Member ...`